bin.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+const config = require('config')
+const { SeedingServer } = require('.')
+const App = require('./src/app.js')
+const path = require('path')
+
+main()
+
+async function main() {
+  const opts = {
+    dbName: config.get('store.dbName')
+  }
+
+  const storage = config.get('store.path');
+  opts.storage = storage && path.join(__dirname, storage)
+
+  try {
+    opts.topic = Buffer.from(config.get('hyperswarm.topicKey'), 'hex')
```

Comment:
dont call it hyperswarm, call it somethink like slashtags.topickey

Reply:
Ok, but note that you will need to update the config at the serve side, otherwise it won't work.